packages/cli-build/src/finalize.js

```diff
@@ -19,7 +19,7 @@ export const finalize = command('finalize', {
   log.info('Finalizing parallel build...');
 
   // rely on the parallel nonce to cause the API to return the current running build for the nonce
-  let { data: build } = await percy.client.createBuild();
+  let { data: build } = await percy.client.createBuild({ execType: percy.execType });
```

Comment:
@itsjwala As we don't have build finalize support for app. Do we really need this?
cc @ninadbstack

Reply:
This will depend on what API defaults to if its `web` then you can only change part which is `app:*` related

Reply:
passing `execType` in createBuild in finalize call.